common/src/types.rs

```diff
@@ -8,6 +8,7 @@ pub type Timestamp = i64;
 pub type BlockHeight = u64;
 pub type ConsensusRound = u64;
 pub type FinalizationProof = Vec<TypedSignature<BlockHeader>>;
+pub type TxDelegateProof = (PublicKey, PublicKey, bool, BlockHeight);
```

Comment:
I think the variable name is misleading because signature is the proof.

Reply:
Changed the name to `DelegationTransactionData` and it is wrapped into `DelegationSignature` which is `TypedSignature<DelegationTransactionData>`